src/lando/middleware.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.http import HttpRequest, HttpResponse
+
+
+class ResponseHeadersMiddleware:
+    """Add custom response headers for each request."""
+
+    def __init__(self, get_response: callable):
```

Comment:
Type hints for callable functions don't support the `callable` builtin function, since it is just a function and not an actual type like `list`/`dict`/`int` etc. You need to use `collections.abc.Callable` instead.

Nit: you could also add the parameter and return types since the `__call__` function below implies they are known. 
```suggestion
    def __init__(self, get_response: Callable[[HttpRequest], HttpResponse]):
```